### ISSUE TITLE
🔴 You cannot delete your account (lol)

### ISSUE THREAD

**Issue by emmvs** [emmvs/five_things#134]
<img width="367" height="251" alt="Image" src="https://github.com/user-attachments/assets/e0ddb975-6d3c-4def-b63d-f65876c926d9" />

http://localhost:3000/users/edit
